opencti-platform/opencti-front/src/components/dataGrid/components/DataTableHeaders.tsx

```diff
@@ -48,133 +44,134 @@ const DataTableHeaders: FunctionComponent<DataTableHeadersProps> = ({
-              {activeColumn?.isSortable && (<MenuItem onClick={() => onSort(activeColumn.id, false)}>{t_i18n('Sort Desc')}</MenuItem>)}
-              {(activeColumn && availableFilterKeys?.includes(activeColumn.id)) && (
-                <MenuItem
-                  onClick={() => {
-                    onAddFilter(activeColumn.id);
-                    handleClose();
-                  }}
-                >
-                  {t_i18n('Add filtering')}
-                </MenuItem>
-              )}
-            </Menu>
-          )}
-          {effectiveColumns
-            .filter(({ id }) => !['select', 'navigate'].includes(id))
-            .map((column) => (
-              <DataTableHeader
-                key={column.id}
-                column={column}
-                setAnchorEl={setAnchorEl}
-                isActive={activeColumn?.id === column.id}
-                setActiveColumn={setActiveColumn}
-                setLocalStorageColumns={setLocalStorageColumns}
-                containerRef={containerRef}
-                sortBy={sortBy === column.id}
-                orderAsc={!!orderAsc}
-              />
-            ))}
-          <span style={{ width: 'calc( 42px + 12px )' }} />
-        </>
-      )}
+                      setColumns(newColumns);
+                    }}
+                  >
+                    <Droppable droppableId="droppable-list">
+                      {(provided) => (
+                        <div ref={provided.innerRef} {...provided.droppableProps}>
+                          {draggableColumns.map((c, index) => (
+                            <Draggable
+                              key={index}
+                              draggableId={c.id}
+                              index={index}
+                            >
+                              {(item) => (
+                                <MenuItem
+                                  ref={item.innerRef}
+                                  {...item.draggableProps}
+                                  {...item.dragHandleProps}
+                                >
+                                  <DragIndicatorOutlined fontSize="small" />
+                                  <Checkbox
+                                    onClick={() => handleToggleVisibility(c.id)}
+                                    checked={c.visible}
+                                  />
+                                  {c.label}
+                                </MenuItem>
+                              )}
+                            </Draggable>
+                          ))}
+                          {provided.placeholder}
+                        </div>
+                      )}
+                    </Droppable>
+                  </DragDropContext>
+                )}
+                {/* <MenuItem onClick={() => handleToggleVisibility(column.id)}>{t_i18n('Hide column')}</MenuItem> */}
```

Comment:
to uncomment?

Reply:
Was there before, not sure, I will delete the line